integr8ly/tutorial-web-app
# INTLY-2663 add user sso provision to startup loading task

verification:
- an integreatly cluster on openshift 3 is required
- ensure the user-rhsso clusterserviceclass exists by searching for
  it on the main page (service catalog)
- ensure USER_SSO_URL is set on the managed service broker env
  var
- run this branch locally against the cluster
- ensure that the user sso is now also provisioned at startup
- ensure the link on the right-hand side works as expected
